wikipedia-sentences/README.md

```diff
@@ -1,50 +1,79 @@
+
 # Semantic Wikipedia Search with Transformers and DistilBERT
 
 ![](https://docs.jina.ai/_images/jinabox-wikipedia.gif)
 
-| item   | content                                          |
-|--------|--------------------------------------------------|
-| Input  | 1 text file with 1 sentence per line             |
-| Output | *top_k* number of sentences that match input query |
 
-This is an example of using [Jina](http://www.jina.ai)'s neural search framework to search through a [selection of individual Wikipedia sentences](https://www.kaggle.com/mikeortman/wikipedia-sentences) downloaded from Kaggle. It uses the [`distilbert-based-uncased`](https://huggingface.co/distilbert-base-uncased) language model from [Transformers](https://huggingface.co).
+## Table of contents: 
 
-## 🐳 Run in Docker
+- [Overview](#overview)
+- [🐍 Build the app with Python](#---build-the-app-with-python)
+- [🔮 Overview of the files in this example](#---overview-of-the-files-in-this-example)
+- [🌀 Flow diagram](#---flow-diagram)
+- [🔨 Next steps, building your own app](#---next-steps--building-your-own-app)
+- [🐳 Deploy the prebuild application using Docker](#---deploy-the-prebuild-application-using-docker)
+- [🙍 Community](#---community)
+- [🦄 License](#---license)
 
-To test this example you can run a Docker image with 30,000 pre-indexed sentences:
+# Overview
```

Comment:
All of these headings should be `##` level, not `#` level. Otherwise they're enormous

Reply:
fixed.